starknet/src/utils/endian.cairo

```diff
@@ -37,7 +37,9 @@ fn into_le_u64_array(self: Array<u256>) -> (Array<u64>, u64) {
                 out.append(low_high);
                 out.append(low_low);
             },
-            Option::None => {}
+            Option::None => {
+                break (0);
```

Comment:
shouldn't we panic here?

Reply:
i don't think so: if it's `None` it means `self` is empty.
we will return `out` which will be empty also, so we've turned an empty array into another empty array, which sounds fine to me?

Reply:
okay yeah its probably okay then. 

Although this function has a specific usecase and should never be passed with an empty array.  The auditors also recommended to throw an error so might as well just to be safe.

Reply:
Ok sir, your wish is my command

fd602371d5c0aafde565efd771d1b59685294599